Helper/Data.php

```diff
@@ -153,9 +155,19 @@ protected function build_params($params) {
         return 'data=' . urlencode(base64_encode(json_encode($params)));
     }
 
+    protected function unwrap_params($params) {
```

Comment:
could you describe what you are doing here?

Reply:
Just trying to get it in a readable format to log into the Klaviyo.log file if any failures occur on the track call